grammar.js

```diff
@@ -3,7 +3,12 @@
 module.exports = grammar({
   name: "aiken",
   rules: {
-    source_file: ($) => repeat($._definition),
+    source_file: ($) => repeat(choice(
+      $._definition,
+      $.module_comment,
+      $.statement_comment,
```

Comment:
Can we rename this to doc_comment or definition_comment? We don't use the word statement anywhere in the main compiler and I'd like to be consistent with naming.

Reply:
Can do!